velero-plugins/imagestream/shared.go

```diff
@@ -1,13 +1,34 @@
 package imagestream
 
 import (
+	"context"
 	"errors"
+
 	"github.com/containers/image/v5/types"
+	"github.com/openshift/client-go/route/clientset/versioned/scheme"
+	routev1client "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
+	velero "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
+var (
+	internalRegistrySystemContextVar *types.SystemContext
+	oadpRegistryRoute *string
```

Comment:
As with common, I'd rather see this added to the BackupPlugin and RestorePlugin structs than use a global var.